py/jpy-integration/build.gradle

```diff
@@ -82,54 +83,129 @@ idea {
   }
 }
 
-// only enable this if py building is enabled
-if (PyEnv.pythonEnabled(project)) {
+Task testJavaToPython = tasks.create 'testJavaToPython', {
+  Task t ->
+    t.group = 'python'
+    t.description = 'Run the java test suite that depends on deephaven-jpy (org.jpy.PyLib, etc)'
+    check.dependsOn t
+}
```

Comment:
This task doesn't seem to depend on anything else, so is a no-op?

Reply:
Correct, as in my comment above, this will actually do something once i have gradle calling docker calling gradle calling junit calling python